### PR TITLE
remove result, update desciptions

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe: LGTY? 
- removed redundant response variables
- updated descriptions for response variables
- added method to convert address_error_status to array
- first and last name no longer required
- removed extraneous external libraries

Just lmk if you have any questions. Some of these updates reflect further discussions had today.

### CODE REVIEW

File: src/outbound.coffee [hunk 1]
```diff
@@ -83,16 +79,15 @@ response = (vars, req, res) ->
   outboundRes = {}
 
   if res.status is 200
-    outboundRes.address_error_status = body.StandardizedAddress?.ADDRESS_ERROR_STATUS or null
-    outboundRes.address_code = body.StandardizedAddress?.ADDRESS_CODE or null
-    outboundRes.result_code = body.Message?.RESULT_CODE or null
-    outboundRes.match_type = body.Stage2Data?.Match_Type or null
+    outboundRes.address_error_status = parseAddressErrorStatus(body.StandardizedAddress?.ADDRESS_ERROR_STATUS)
```
Comment (alexkwolfe) [inline src/outbound.coffee]: IMO, this would make more sense as a plural noun like `address_error_codes`.

Comment (ros3bud) [inline src/outbound.coffee]: Makes sense - especially given the new parsing.

File: src/outbound.coffee [hunk 2]
```diff
@@ -101,14 +96,11 @@ response = (vars, req, res) ->
...
-    { name: 'zip4_credit_score', type: 'integer', description: 'Experian generated Median credit score based on address' }
-    { name: 'address_error_status', type: 'string', description: 'Experian generated code up to 30 characters, 2-byte codes placed left to right' }
-    { name: 'address_code', type: 'string', description: 'Experian generated code for additional information about address' }
-    { name: 'result_code', type: 'string', description: 'Experian generated 4 byte error code for additional information about lead data' }
-    { name: 'match_type ', type: 'string', description: 'Experian generated indicator of match level. E = Median level, F = Household Level, I = Person Level, N = No Match' }
+    { name: 'outcome', type: 'string', description: 'The outcome of the lookup will be success if a credit score is returned, and failure if not. An outcome of Error indicates a problem was encountered during lookup.' }
+    { name: 'reason', type: 'string', description: 'The description for a failure or error outcome.' }
+    { name: 'zip4_credit_score', type: 'string', description: 'Experian generated median credit score.' }
```
Comment (alexkwolfe) [inline src/outbound.coffee]: Make a `number` type.

File: src/outbound.coffee [hunk 3]
```diff
@@ -101,14 +96,11 @@ response = (vars, req, res) ->
...
+    { name: 'address_error_status', type: 'array', description: 'Experian generated codes indicating error information about submitted address data. Codes are 2 digits each and there can be up to 15.' }
+    { name: 'match_type', type: 'string', description: 'Experian generated indicator of match level. Possible outcomes are: Median level, Household Level, Person Level, or No Match' }
```
Comment (alexkwolfe) [inline src/outbound.coffee]: Capitalization in "Median level"

**Comment by alexkwolfe** [comment]
LGTM

**Pull request merged**